pkg/forwarder/transaction_container.go

```diff
@@ -64,6 +74,7 @@ func (f *transactionContainer) Add(t Transaction) (int, error) {
 		}
 		if diskErr != nil {
 			diskErr = fmt.Errorf("Cannot store transactions on disk:%v", diskErr)
+			f.telemetry.addErrorsCount()
```

Comment:
nit: `incErrorsCount` would be a better naming, add is often used with an integer parameter.

Reply:
Code Updated.